### ISSUE TITLE
Send a private notice every X minutes

### ISSUE THREAD

**Issue by strife84** [znc/znc#1618]
Hi, I'm new to the Znc world and I would like to know if there's a way to send a private notice to a specific user every X minutes. thank you

**Comment by teward** [comment]
There is no way out of the box.  You would need to design a module that would be embedded in ZNC that could do this.  (There is also no current module for this).

If I may ask, though, what's the use-case for this?  It sounds like this is just an abuseable module / attack vector against other IRC users from my perspective as an IT Security person.

**Comment by strife84** [comment]
I'm sorry if I gave you a bad impression! I bought a vps and installed a znc on it to chat with my friends on irc. I've also installed the znc-push module (found it here on github) so I can receive a notification on telegram when someone types my nickname or send me a private notification/private message on irc. Sometime I don't receive anything so I think the znc got disconnected but  when I connect to the channel I just find nobody typed my nickname. The znc-push module and telegram combination works only with private messages/notificatons and when someone writes my nick on the channel, so I thought it would have been a good idea to make the znc send a private notification to myself every 30 minutes so I can see the message on telegram (something like "/notice myself hey, you are still online"). sorry again for the misunderstanding. also, english is not my first english, sorry for this too.
